### PR TITLE
Admindashboard/bugs

### PR DESCRIPTION
**Pull request by expertdeveloper5821**
1>  After Deleting the data of the rooms still delete pop up is showing.
2> the delete functionality is not working Properly in the rooms.
3> Toggle Menu is not visible on on admin dasboard on sepactator module

### CODE REVIEW

File: src/Components/CommonComponent/Table/Table.tsx [hunk 1]
```diff
@@ -74,6 +74,7 @@ const TableData = ({ data, columns, deleteroom, type, handleEdit }: TablePropsTy
     }
   };
 
+  console.log('inside table data ==>', data);
```
Comment (Ashok-tgc) [inline src/Components/CommonComponent/Table/Table.tsx]: remove console.log

File: src/Components/HOC/AuthHoc.tsx [hunk 2]
```diff
@@ -14,6 +21,14 @@ const AuthHoc = ({ children, heading, subheading }) => {
           <p className={styles.subheading}>{subheading}</p>
         </div>
         <div className={styles.formWrapper}>{children}</div>
+        {stepperIsVisible && (
+          <div className={step === 3 ? `${styles.flexGap} ${styles.flex_margin}` : styles.flexGap}>
```
Comment (Ashok-tgc) [inline src/Components/HOC/AuthHoc.tsx:26]: using same styles for step = 1,2,3,4?

Comment (expertdeveloper5821) [reply src/Components/HOC/AuthHoc.tsx:26]: done.

File: src/Components/CommonComponent/Table/Table.tsx (hunk 1 above)
Comment (expertdeveloper5821) [reply src/Components/CommonComponent/Table/Table.tsx]: done.

File: src/Components/HOC/AuthHoc.tsx (hunk 2 above)
Comment (expertdeveloper5821) [reply src/Components/HOC/AuthHoc.tsx:26]: yes using same styles, now used map instead of hardcoding.

**Pull request merged**